src/rules/no_var.rs

```diff
@@ -43,11 +63,9 @@ impl<'c> Visit for NoVarVisitor<'c> {
 
   fn visit_var_decl(&mut self, var_decl: &VarDecl, _parent: &dyn Node) {
     if var_decl.kind == VarDeclKind::Var {
-      self.context.add_diagnostic(
-        var_decl.span,
-        "no-var",
-        "`var` keyword is not allowed",
-      );
+      self
+        .context
+        .add_diagnostic(var_decl.span, "no-var", MESSAGE);
```

Comment:
Could you make `"no-var"` const just like `MESSAGE`?  Here's an example: https://github.com/denoland/deno_lint/blob/master/src/rules/no_eval.rs#L17